src/script/conversation/ConversationMapper.ts

```diff
@@ -393,7 +402,8 @@ export class ConversationMapper {
 
         return mergedConversation;
       })
-      .concat(failedConversations);
+      .concat(failedConversations as ConversationDatabaseData[])
```

Comment:
Don't do this please. Use type guards to make sure the data is the right one. Assertions are bad :)

Reply:
gone